code-challenges.js

```diff
@@ -26,20 +26,45 @@ const fruit2 = "banana"
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+let combinedArrays = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+console.log(combinedArrays.length)
 // Expected output: 9
 
 // Pseudo code:
+// Create a variable that will hold both arrays together as one with .concat()
+// add both array lengths together padres1998WorldSeriesRuns & adres1984WorldSeriesRuns using .concat()and save that value into my created variable
```

Comment:
This can be explained better, as .concat isn't adding the lengths together, it is concatenating two arrays together.